terraform/internal/azure/vm/output.tf

```diff
@@ -39,6 +39,10 @@ locals {
       </Query>
       </QueryList>
       "@
+
+      # Monitor the initialization scripts. These can take several minutes to complete.
+      while(1){Get-ScheduledTask -TaskPath "\Rancher\Terraform\"; sleep 5; clear}
+
```

Comment:
Q: Do we need a timeout to prevent infinite waiting?

Reply:
No, this is just a utility command that doesn't get used when running the terraform. I added it as I was frequently using it to determine when I could rdp into the node and start working in it.